vendor/kcfinder/core/bootstrap.php

```diff
@@ -108,6 +108,9 @@ function checkLSSession()
     {
         define("BASEPATH", realpath($sLimesurveyFolder . "/../vendor/yiisoft/yii/framework") . "/");
     }
+
+    include APPPATH . '../setdebug.php';
```

Comment:
Hacking into vendor/ folder???

Reply:
Well, that file is already hacked :)
It is not KCFinder original file